README.md

```diff
@@ -5,7 +5,7 @@ The setup files add some functionality I find important. Things such as a fuzzy
 ## Install
 ### Linux (Ubuntu)
   - Clone repo
-  - Run `bash-setup.sh`
+  - Run `bash-setup.sh your_email your_username ssh_key_name gh_auth_token_location gpg_key_location`
```

Comment:
This seems a bit clunky. Might be worth looking into if this is worth completely doing upfront like this, & if it's possible to shift off some of the hard work to environment variables, prompted user input, or hard coding.